app/queries/consumer_apps/find_or_create_all_query.rb

```diff
@@ -5,7 +5,8 @@ def self.call
       existing_platforms = ConsumerApp.where(app_bundle: forem_bundle).pluck(:platform)
       (ConsumerApp::FOREM_APP_PLATFORMS - existing_platforms).each do |platform|
         # Re-create the supported Forem apps if they're missing
-        ConsumerApp.create(app_bundle: forem_bundle, platform: platform, active: true)
+        ConsumerApp.create(app_bundle: forem_bundle, platform: platform, active: true,
+                           team_id: ConsumerApp::FOREM_TEAM_ID)
```

Comment:
We want these queries to start creating new instances the Forem Consumer Apps with the `team_id`